### PR TITLE
Edit "wolfdancers" external contribution

### PR DESCRIPTION
**Pull request by DonNateR**
- Changes to PR #1269

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b6bd16cbc8/how-to/).

File: content/rackspace-metrics/integrate-statsd-with-rackspace-metrics.md [hunk 1]
```diff
@@ -5,50 +5,60 @@ title: Integrate statsD with Rackspace Metrics
 type: article
```
Comment (cyrichardson) [inline content/rackspace-metrics/integrate-statsd-with-rackspace-metrics.md]: Need to change statsD in title to StatsD to match rest of article.

File: content/rackspace-metrics/integrate-statsd-with-rackspace-metrics.md [hunk 2]
```diff
@@ -5,50 +5,60 @@ title: Integrate statsD with Rackspace Metrics
...
--   Install the Rackspace Metrics statsd backend engine
--   Integrate with Rackspace Metrics
-
-### Install the Rackspace Metrics statsd backend engine
-
-1. Clone the statsd repository by typing the following at a command-line
-interface:
-
-        git clone https://github.com/etsy/statsd.git
+This article describes how to configure StastD to send metrics to Rackspace Metrics. If you don't have StatsD installed, follow the instructions on the[StatsD Website](https://github.com/etsy/statsd/blob/master/README.md).
```
Comment (cyrichardson) [inline content/rackspace-metrics/integrate-statsd-with-rackspace-metrics.md]: Need a space between "the" and "[StatsD"  and change "Website" to "website".

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b9148bfd48/how-to/)

**Comment by cyrichardson** [comment]
LGTM. Merging.

**Pull request merged**